Gemfile

```diff
@@ -3,13 +3,18 @@ git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
 ruby '3.2.2'
 
+gem 'activestorage-validator'
 gem 'bootsnap', require: false
 gem 'bootstrap', '~> 5.3.1'
+gem 'bcrypt', '~> 3.1.7'
+gem 'cancancan'
```

Comment:
Are we implementing cancancan abilities in this PR? If not, lets remove this gem. Let's add it back when we are introducing the functionalities.